### DISCUSSION TITLE
Iframe and Webview support

### DISCUSSION THREAD

**Discussion by burakakca** [facebook/react-strict-dom#252] [General]
What you guys think that supporting both of them?

**Comment by necolas** [comment]
What did you have in mind?

**Reply by nmn** [reply]
I think an interesting idea here would be to add an `html.iframe` component that renders a webview within React Native.

This would also be similar to Expo web components.

However, we would need to figure out the bits of the iframe API that would need to be polyfilled.

**Reply by burakakca** [reply]
yeah, exactly

**Reply by necolas** [reply]
React Native hasn't had a WebView export for years. I don't think we have a use case for this internally. It's probably something you can roll yourself for now
